src/openklant/conf/base.py

```diff
@@ -1,448 +1,40 @@
-# https://docs.djangoproject.com/en/2.0/topics/i18n/
-
-LANGUAGE_CODE = "nl"
-
-TIME_ZONE = "UTC"
-
-USE_I18N = True
-
-USE_L10N = True
-
-USE_TZ = True
-
-USE_THOUSAND_SEPARATOR = True
-
-#
-# DATABASE and CACHING setup
-#
-DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.postgresql",
-        "NAME": config("DB_NAME", "openklant"),
-        "USER": config("DB_USER", "openklant"),
-        "PASSWORD": config("DB_PASSWORD", "openklant"),
-        "HOST": config("DB_HOST", "localhost"),
-        "PORT": config("DB_PORT", 5432),
-    }
-}
-
-# keep the current schema for now and deal with migrating to BigAutoField later, see
-# https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-DEFAULT_AUTO_FIELD
-DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
-
-CACHES = {
-    "default": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_DEFAULT', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "axes": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_AXES', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-    "oidc": {
-        "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://{config('CACHE_OIDC', 'localhost:6379/0')}",
-        "OPTIONS": {
-            "CLIENT_CLASS": "django_redis.client.DefaultClient",
-            "IGNORE_EXCEPTIONS": True,
-        },
-    },
-}
 
+init_sentry()
 
 #
 # APPLICATIONS enabled for this project
 #
-
-INSTALLED_APPS = [
-    # Note: contenttypes should be first, see Django ticket #10827
-    "django.contrib.contenttypes",
-    "django.contrib.auth",
-    "django.contrib.sessions",
-    # NOTE: If enabled, at least one Site object is required and
-    # uncomment SITE_ID above.
-    "django.contrib.sites",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    # Admin auth
-    # "django_otp",
-    # "django_otp.plugins.otp_static",
-    # "django_otp.plugins.otp_totp",
-    # "two_factor",
-    # Optional applications.
-    "ordered_model",
-    "django_admin_index",
-    "django.contrib.admin",
-    # 'django.contrib.admindocs',
-    # 'django.contrib.humanize',
-    # 'django.contrib.sitemaps',
-    # External applications.
-    "axes",
-    "corsheaders",
-    "vng_api_common",  # before drf_yasg to override the management command
-    "privates",
-    "drf_spectacular",
-    "mozilla_django_oidc",
-    "mozilla_django_oidc_db",
-    "rest_framework",
-    "django_markup",
-    "solo",
-    "django_jsonform",
-    "log_outgoing_requests",
+# FIXME implement 2fa
+INSTALLED_APPS.pop(INSTALLED_APPS.index("maykin_2fa"))
+INSTALLED_APPS = INSTALLED_APPS + [
     # Project applications.
-    "openklant",
```

Comment:
Removed and missing from open-api-framework

Reply:
I keep thinking this is OpenZaak not OpenKlant.
This should not have been there in the first place, I dont think.

Reply:
For some reason adding this breaks things when running tests:

```
django.core.exceptions.ImproperlyConfigured: '/home/steven/Documents/open-klant/src/openklant/fixtures' is a default fixture directory for the 'openklant' app and cannot be listed in settings.FIXTURE_DIRS.
```